packages/shortest/src/cli/commands/init.ts

```diff
@@ -43,84 +62,256 @@ initCommand
     });
   });
 
+interface Ctx {
+  alreadyInstalled: boolean;
+  anthropicApiKeyExists: boolean;
+  anthropicApiKeyName: string;
+  anthropicApiKeyValueNeeded: boolean;
+  anthropicApiKeyValue: string;
+  envFile: EnvFile;
+  generateSampleTestFile: boolean;
+  supportedFrameworkInfo: FrameworkInfo | null;
+  shortestLoginEmail: string;
+  shortestLoginPassword: string;
+}
+
 export const executeInitCommand = async () => {
-  const tasks = new Listr(
+  const tasks = new Listr<Ctx>(
     [
       {
-        title: "Checking for existing installation",
-        task: async (ctx, task): Promise<void> => {
-          const packageJson = await getPackageJson();
-          ctx.alreadyInstalled = !!(
-            packageJson?.dependencies?.["@antiwork/shortest"] ||
-            packageJson?.devDependencies?.["@antiwork/shortest"]
-          );
-          if (ctx.alreadyInstalled) {
-            task.output = `Shortest is already installed`;
-          } else {
-            task.output = "Shortest is not installed, starting installation.";
-          }
-        },
-        rendererOptions: {
-          persistentOutput: true,
-        },
-      },
-      {
-        title: "Installing dependencies",
-        enabled: (ctx): boolean => !ctx.alreadyInstalled,
-        task: async (_, task): Promise<Readable> => {
-          const installCmd = await getInstallCmd();
-          task.title = `Executing ${installCmd.toString()}`;
-          return spawn(installCmd.cmd, installCmd.args).stdout;
-        },
-        rendererOptions: {
-          persistentOutput: true,
-        },
-      },
-      {
-        title: `Creating ${CONFIG_FILENAME}`,
-        enabled: (ctx): boolean => !ctx.alreadyInstalled,
-        task: async (_, task) => {
-          const configPath = join(process.cwd(), CONFIG_FILENAME);
-          const exampleConfigPath = join(
-            fileURLToPath(new URL("../../src", import.meta.url)),
-            `${CONFIG_FILENAME}.example`,
-          );
-
-          const exampleConfig = await readFile(exampleConfigPath, "utf8");
-          await writeFile(configPath, exampleConfig, "utf8");
-          task.title = `${CONFIG_FILENAME} created.`;
-        },
+        title: "[🧪 Experimental Next.js] Confirm sample test file generation",
```

Comment:
This introduces the [new MVP setup](https://github.com/antiwork/shortest/discussions/375#discussioncomment-12559992) to the `init` command. 

We need to ask if the user wants to generate the sample file before the installation, so that a different value for `testPattern` can be set (`shortest` folder). If we automate test files generation, I think it would be simpler to just have all the files in a dedicated folder, or conflict with a more generic `test` folder.